### DISCUSSION TITLE
Request For Prisma Migration Failure Recovery "Nuclear Option"

### DISCUSSION THREAD

**Discussion by keepforever** [epicweb-dev/epic-stack#633] [Ideas]
One "gotcha" that happens to me over and over again when working with a new Epic Stack project idea is:

•  I add data to the staging/production volumes
• then incorrectly manage the prisma migration,
•  then, deploy fails
• then I cannot recover because fixing that state is very confusing. 

I'm wondering if there could be some guidance added to docs about a simple change to the github action or elsewhere that would essentially: 

• drop the volume and/or all data
• create fresh volume and attach
• re-init the tables with current `prisma.schema`

I understand this would be very dangerous to make too easy to do, but, if there was someplace in the docs that was like "if you find yourself totally borked, you can swap this line with this and nuke the database".

This would be particularly useful in the early stages of iteration after init of new boiler and you're making big, sweeping changes.

Thanks in advace for the consideration. 

In the meantime, I just literally have to init a new project and try again.

**Comment by kentcdodds** [comment]
Having a document or blog post about what to do when you find yourself in this situation would be very helpful.

Anyone is welcome to write something up :)
